Application/EdFi.Ods.AdminApi/Features/AdminApiResponse.cs

```diff
@@ -37,14 +41,62 @@ public static IResult Ok(T result)
 
     public static IResult Ok(T result, JsonSerializerSettings jsonSerializerSettings)
     {
-        var data = new AdminApiResponse<T>(200, "Request successful", result);
-        var dataSerialize = JsonConvert.SerializeObject(data, Formatting.Indented, jsonSerializerSettings);
-        return Results.Text(dataSerialize, "application/json", System.Text.Encoding.UTF8);
+        var statusCode = HttpStatusCode.OK;
+        var jsonResponse = SerializeObjectAsJson(result, statusCode, "Request successful", jsonSerializerSettings);
+        return new JsonSerializeResult(jsonResponse, statusCode);
     }
 
     public static IResult Created(T result, string name, string getUri)
         => Results.Created(getUri, new AdminApiResponse<T>(201, $"{name} created successfully", result));
+
+    public static IResult Created(T result, string name, string getUri, JsonSerializerSettings jsonSerializerSettings)
+    {
+        var statusCode = HttpStatusCode.Created;
+        var jsonResponse = SerializeObjectAsJson(result, statusCode, $"{name} created successfully", jsonSerializerSettings);
+        return new JsonSerializeResult(jsonResponse, statusCode, getUri);
+    }
 
     public static IResult Updated(T result, string name)
-        => Results.Ok(new AdminApiResponse<T>(200, $"{name} updated successfully", result));
+        => Results.Ok(new AdminApiResponse<T>(200, $"{name} updated successfully", result));
+
+    public static IResult Updated(T result, string name, JsonSerializerSettings jsonSerializerSettings)
+    {
+        var statusCode = HttpStatusCode.OK;
+        var jsonResponse = SerializeObjectAsJson(result, statusCode, $"{name} updated successfully", jsonSerializerSettings);
+        return new JsonSerializeResult(jsonResponse, statusCode);

```

Comment:
@jagudelo-gap 
Can we do default parameter instead of method overloading ?

Reply:
Done